src/javascript/JContent/JContent.utils.js

```diff
@@ -124,6 +124,19 @@ function uppercaseFirst(string) {
     return string.charAt(0).toUpperCase() + string.substr(1);
 }
 
+const findField = (fieldName, types) => {
```

Comment:
I may missing something but how do you know that fieldName exists in the expected type? From my understanding this method will return true if fieldName exists in any type.

Reply:
Don't we need to either check the type in this method, or parameterize the hook to only return fields from a given type?

Reply:
yes, agree - you should always ask for a field in a specific type , not look in all types